hummingbot/market/bamboo_relay/bamboo_relay_market.pyx

```diff
@@ -1315,7 +1316,7 @@ cdef class BambooRelayMarket(MarketBase):
                    object price=s_decimal_NaN,
                    dict kwargs={}):
         cdef:
-            int64_t tracking_nonce = <int64_t>(time.time() * 1e6)
+            int64_t tracking_nonce = get_tracking_nonce()
```

Comment:
i think`<int64_t>` casts the python `int` to cython `int64_t` so we might want to keep it

Reply:
i removed the cast as I was worried about performance, after some researching I don't think it's matter much whether to have explicit casting or implicit casting. I'll add it back it.